src/core/global-api/index.js

```diff
@@ -17,6 +17,10 @@ import {
   defineReactive
 } from '../util/index'
 
+import { default as Dep, pushTarget, popTarget } from '../observer/dep'
+import { afterFlush, forceFlush } from '../observer/scheduler'
+import { default as Watcher } from '../observer/watcher'
```

Comment:
Why are you not using default import syntax?

L22: `import Watcher from '../observer/watcher'`
L20: `import Dep,  { pushTarget, popTarget } from '../observer/dep'`